di-ipv-credential-issuer-stub/src/test/java/uk/gov/di/ipv/stub/cred/fixtures/TestFixtures.java

```diff
@@ -29,9 +24,58 @@ private TestFixtures() {}
     public static final String EC_PUBLIC_JWK_3 =
             "{\"kty\":\"EC\",\"crv\":\"P-256\",\"x\":\"XqrUBZchnNiPxCA4Gwb7EikNGkoOwubgEqC5ZCfOSNI\",\"y\":\"bRnQYD-zp__65OPmvWCCIy-RKfHTqmTk21JlY376N3Q\"}";
 
-    public static final String CLIENT_CONFIG_WITH_PUBLIC_JWK =
-            "ewogICJhVGVzdENsaWVudCI6IHsKICAgICJzaWduaW5nQ2VydCI6ICJUT1RBTExZLUEtQ0VSVElGSUNBVEUiLAogICAgImp3dEF1dGhlbnRpY2F0aW9uIjogewogICAgICAic2lnbmluZ1B1YmxpY0p3ayI6ICJ7XCJrdHlcIjpcIkVDXCIsXCJjcnZcIjpcIlAtMjU2XCIsXCJ4XCI6XCJFOVp6dU9vcWNWVTRwVkI5cnBtVHplemp5T1BSbE9tUEdKSEtpOFJTbElNXCIsXCJ5XCI6XCJLbFRNWnRoSFpVa1l6NUFsZVRROGpmZjBUSmlTM3EyT0I5TDVGdzR4QTA0XCJ9IiwKICAgICAgInZhbGlkUmVkaXJlY3RVcmxzIjogWyJodHRwczovL3Rlc3QtY2xpZW50LmV4YW1wbGUuY29tL2NhbGxiYWNrIl0sCiAgICAgICJhdXRoZW50aWNhdGlvbk1ldGhvZCI6ICJqd3QiCiAgICB9CiAgfQp9";
-
     public static final String DCMAW_VC =
             "eyJ0eXAiOiJKV1QiLCJhbGciOiJFUzI1NiJ9.eyJzdWIiOiJ1cm46dXVpZDo1ZDZkNjgzMy04NTEyLTRlMzctYjVlYS1iZTdkZTc3OTQ4ZGQiLCJhdWQiOiJodHRwczpcL1wvaWRlbnRpdHkuYnVpbGQuYWNjb3VudC5nb3YudWsiLCJuYmYiOjE2OTgxNDU2NjksImlzcyI6Imh0dHBzOlwvXC9kY21hdy1jcmkuc3R1YnMuYWNjb3VudC5nb3YudWsiLCJ2YyI6eyJ0eXBlIjpbIlZlcmlmaWFibGVDcmVkZW50aWFsIiwiSWRlbnRpdHlDaGVja0NyZWRlbnRpYWwiXSwiY3JlZGVudGlhbFN1YmplY3QiOnsibmFtZSI6W3sibmFtZVBhcnRzIjpbeyJ0eXBlIjoiR2l2ZW5OYW1lIiwidmFsdWUiOiJLZW5uZXRoIn0seyJ0eXBlIjoiRmFtaWx5TmFtZSIsInZhbHVlIjoiRGVjZXJxdWVpcmEifV19XSwiYmlydGhEYXRlIjpbeyJ2YWx1ZSI6IjE5NjUtMDctMDgifV0sInBhc3Nwb3J0IjpbeyJleHBpcnlEYXRlIjoiMjAzMC0wMS0wMSIsImRvY3VtZW50TnVtYmVyIjoiMzIxNjU0OTg3In1dfSwiZXZpZGVuY2UiOlt7ImFjdGl2aXR5SGlzdG9yeVNjb3JlIjoxLCJjaGVja0RldGFpbHMiOlt7ImNoZWNrTWV0aG9kIjoidnJpIn0seyJiaW9tZXRyaWNWZXJpZmljYXRpb25Qcm9jZXNzTGV2ZWwiOjMsImNoZWNrTWV0aG9kIjoiYnZyIn1dLCJ2YWxpZGl0eVNjb3JlIjoyLCJzdHJlbmd0aFNjb3JlIjozLCJ0eXBlIjoiSWRlbnRpdHlDaGVjayIsInR4biI6ImMxZjZmMzIwLTAxZWYtNDRhNy1hZTA1LTIyNGU1MzgwZDUwMCJ9XX0sImp0aSI6InVybjp1dWlkOmU5MzdlODEyLWRhZmUtNDJlNC1hMDk0LTZjOWQ0MWZlZTUwYyJ9.3d6uaGQHb5zBkSMvgZ7S58_Y_lk_hSRk7ng5WNRf7O437IJKckvSYP_HNX6rCGOhb4QuW2ROCcfiCxyRO73pOg";
+
+    public static final String CLIENT_CONFIG =
```

Comment:
Could this just be a `ClientConfig` object, to avoid serialising/deserialising? I guess it'll be slightly clunkier in Java syntax :/

If not, I wonder if it'd be cleaner to load this as a JSON resource file, rather than embedding in Java.

Reply:
I think making it an object will be ok - will try it. I changed it from being b64 encoded to make it readable, but should have gone a step further!